tests/conftest.py

```diff
@@ -37,17 +51,17 @@ def restore_config(request, node_config):
 
 @pytest.fixture(scope='module')
 def node_config():
-    return copy.deepcopy(CONFIG)
+    return copy.deepcopy(CONFIG_ED25519)
```

Comment:
Did you remove support for ECDSA? if so, can you please remove `USER_PRIVATE_KEY_ECDSA` and `USER_PUBLIC_KEY_ECDSA`, and all the code related to that?